crates/rover-client/src/operations/subgraph/fetch_all/fetch_all_query.graphql

```diff
@@ -0,0 +1,14 @@
+query SubgraphFetchAllQuery($graph_ref: ID!) {
+  variant(ref: $graph_ref) {
+    __typename
+    ... on GraphVariant {
+      subgraphs {
+        name
+        url
+        activePartialSchema {
+          sdl
+        }
+      }
+    }
+  }
+}
```

Comment:
does this query support pagination? I'm a hair worried about getting the sdl for, say, hundreds of subgraphs in one go

Reply:
Had a look in studio and I can't see that it does, I think the only way we're going to know though is to try it with a big number of graphs and see how it performs. Do we have the power to point it at Indeed's subgraphs, since all we're doing is composing and we have the schemas I don't see why not right?